tests/test_hooks/test_param_scheduler_hook.py

```diff
@@ -74,6 +76,72 @@ def test_after_epoch(self):
         scheduler2.step = Mock()
         scheduler2.by_epoch = True
         runner.param_schedulers = dict(key1=[scheduler1], key2=[scheduler2])
+        hook.after_train_iter(runner, 0)
```

Comment:
Is it necessary to call this method here?

Reply:
I will del it.